resources/property.py

```diff
@@ -57,6 +56,27 @@ def post(self, id):
 
         return property.json(), 201
 
+class ArchiveProperties(Resource):
+    parser = reqparse.RequestParser()
+    parser.add_argument('ids', action='append')
+
+    @admin_required
+    def post(self):
+        data = ArchiveProperties.parser.parse_args()
+        propertyList = []
+
+        for id in data['ids']:
+            property = PropertyModel.find_by_id(id)
+            if(not property):
+                return{'message': 'Properties cannot be archived'}, 400
+            property.archived = True
+            propertyList.append(property)
+
+        for p in propertyList:
+            p.save_to_db()
+
+        return {'properties': [p.json() for p in PropertyModel.query.all()]}, 201
```

Comment:
We are not creating anything here so I think a 200 response is more appropriate.

Reply:
Sure thing. I changed it